week-14/practices/12-context-practices/practice-1-README.md

```diff
@@ -0,0 +1,110 @@
+# Practice: Context Provider with Dynamic Value Practice
+
+In this practice, you will:
+
+* Set up a context provider to use state to hold a dynamic value for the context
+
+## Phase 0: Setup
+
+Download the [starter][context-starter] file for this practice.
+
+Open the starter in VSCode and run `npm install` in the terminal.
+
+Run `npm start` to make sure your project is set up correctly.
+
+
+## Phase 1: Create context
+
+To start, in your `src` directory, create a new directory called `context`. In this
+`context` directory, create a file called `HoroscopeContext.js`. This is where
+all your horoscope context will be placed.
+
+At the top of this file, bring in `createContext` from `react` and create your
+context called `HoroscopeContext`. Make sure to `export` it.
+
+Next, create a new component called `HoroscopeProvider`, that will take `props`.
```

Comment:
This might be a big clearer: take in `props` as its parameter.

Reply:
Yass! Thank youu